schema/sql/api.sql

```diff
@@ -31,6 +31,17 @@ grant usage on sequence "user_id" to "api-user-create";
 grant "api-user-create" to "api";
 grant "api-anon" to "api-user-create";
 
+create role "api-user-update";
+comment on role "api-user-update" is
+    $$Intended for use with user self-management systems$$;
+grant usage on sequence "user_id" to "api-user-create";
+grant select,insert,update,delete on table
+    public."hosts",
+    public."passwd",
```

Comment:
This probably only needs `update` if the role is named correctly

Reply:
I don't know what you mean by this

Reply:
Why are you giving something called "manage" the ability to insert new users?

Reply:
Because the application will only have one credential, and that credential needs to be able to both insert and update users, aka manage them.